app/views.py

```diff
@@ -0,0 +1,104 @@
+from app import app
+from flask import render_template, request, redirect, url_for
+from flask_bootstrap import Bootstrap5
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+Bootstrap5(app)
+
+headers = {
+    "accept": "application/json",
+    "Authorization": f"Bearer {os.getenv('ACCESS_TOKEN_AUTH')}",
+}
+
+MOVIE_DB_IMAGE_URL = "https://image.tmdb.org/t/p/w500"
+
+
+@app.route("/")
+def home():
+    from app.database import Movie, db
```

Comment:
Why under every route is `from app.database import Movie, db` ? Did you have some circular dependency?

Reply:
yep, I've had it all the time. I've added Blueprints and avoided circular imports so I deleted this line of code from every route.